resources/public/api/conf/1.0/application.yaml

```diff
@@ -204,22 +195,104 @@ paths:
             application/json:
               schema:
                 allOf:
-                - $ref: '#/components/schemas/individualPayeTestSupportSaTaxAnnualSummaryResponse'
-              example:
-                pensionsAnnuitiesAndOtherStateBenefits:
-                  otherPensionsAndRetirementAnnuities: 36.5
-                  incapacityBenefit: 980.45
-                refunds:
-                  taxRefundedOrSetOff: 325
-                employments:
-                - employerPayeReference: 123/AB456
-                  taxTakenOffPay: 890.35
-                - employerPayeReference: 456/AB456
-                  taxTakenOffPay: 224.99
+                  - $ref: '#/components/schemas/individualPayeTestSupportSaBenefitsAnnualSummaryResponse'
+              examples:
+                happy-path-1:
+                  $ref: '#/components/examples/taxResponseHappyPath1'
+                happy-path-2:
+                  $ref: '#/components/examples/taxResponseHappyPath2'
       deprecated: false
       security:
-        - application-restricted: []
+        - application-restricted: [ ]
 components:
+  examples:
+    benefitsRequestHappyPath1:
+      description: Payload for scenario HAPPY_PATH_1
+      value:
+        scenario: HAPPY_PATH_1
+    benefitsRequestHappyPath2:
+      description: Payload for scenario HAPPY_PATH_2
+      value:
+        scenario: HAPPY_PATH_2
```

Comment:
Can you please rename `benefitsRequestHappyPath1` and `benefitsRequestHappyPath2` to `requestHappyPath1` and `requestHappyPath2` to make it reusable. As the others have been included now and are all the same, this can be generic and reused to replace all the individual `..requestHappyPath1` and `....requestHappyPath2`. Please delete all the others in this `examples` section when this has been done, thanks.